wiki/Beatmap/Online_offset/id.md

```diff
@@ -1,12 +1,13 @@
 ---
 tags:
   - online song offset
-outdated: true
-outdated_since: 2a73fa15eba9ab6f0c0fbe547d71933628aedc64
+  - offset lagu daring
 ---
 
 # Online offset
 
+*Untuk penggunaan lainnya, lihat [Offset (disambiguasi)](/wiki/Disambiguation/Offset).*
+
 **Online offset** adalah sebuah offset yang diaplikasikan kepada beatmap setelah di ranked. Ini biasanya ketika sebuah beatmap membutuhkan sedikit perubahan timing untuk menyesuaikannya dengan musiknya. Nilai online offset ditentukan dari universal offset top player dan nilai lokal offset.
 
 Ini dapat diubah oleh anggota dari [Nomination Assesment Team](/wiki/People/The_Team/Nomination_Assessment_Team), dan beberapa pengguna, seperti [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) dan [Developers](/wiki/People/The_Team/Global_Moderation_Team). Ini juga artinya online offset tidak dapat dilihat oleh player normal pada situs, namun dapat dilihat di dalam gim. Inilah tampilan yang diperlihatkan bagaimana pengguna yang diperbolehkan untuk melihat dan mengubahnya:
```

Comment:
i found a typo here when rechecking all the stuffs btw, `Assesment` should be `Assessment`

i know this is not included in the intended scope of the pr but would be nice if you can get this fixed as well i suppose 👍

Reply:
I think I should also double-check the entire translation, not just updating it by reference. As another example, `Beatmap Nominators` > `Beatmap Nominator`, `Developers` > `Developer`, etc. I will check it out when I wake up later. Thank you for checking!

Reply:
okok feel free to poke me or the other contributors again later on as always 🧙